atlassian_jwt_auth/contrib/server/helpers.py

```diff
@@ -32,25 +31,34 @@ def _requires_asap(verifier, auth, parse_jwt_func, build_response_func,
         if verify_issuers_func is not None:
             verify_issuers_func(asap_claims, issuers)
         asap_claim_holder.asap_claims = asap_claims
-    except requests.exceptions.HTTPError as e:
+    except PublicKeyRetrieverException as e:
+        if e.status_code not in (403, 404):
```

Comment:
This can be made into a set.

Reply:
For small collections, there's no meaningful difference between tuples and sets:
```
In [86]: %timeit 403 not in (403, 404)
10000000 loops, best of 3: 20.7 ns per loop

In [87]: %timeit 403 not in {403, 404}
10000000 loops, best of 3: 21.2 ns per loop

In [88]: %timeit 404 not in (403, 404)
10000000 loops, best of 3: 27.8 ns per loop

In [89]: %timeit 404 not in {403, 404}
10000000 loops, best of 3: 28.5 ns per loop

In [90]: %timeit 503 not in (403, 404)
10000000 loops, best of 3: 34.2 ns per loop

In [91]: %timeit 503 not in {403, 404}
10000000 loops, best of 3: 34 ns per loop
```

Reply:
Sure. I don't mind.